### ISSUE TITLE
Pre commit broken.

### ISSUE THREAD

**Issue by hbowden** [appleboy/react-recaptcha#219]
I'm trying to create a few pull requests but the pre-commit hook is broken for me. I looked in the package.json and I see the pre-commit runs lint and test. I ran those commands individually and they both pass but I can't commit with git in this repo. Below is the results of running lint and test then trying to commit. The very bottom of the image is the error message I am receiving.

![pre-commit-hook](https://user-images.githubusercontent.com/12447474/35163838-a8348af4-fd0d-11e7-94a9-5fdc84aa9039.png)

It appears that pre-commit is broken on Windows: https://github.com/observing/pre-commit/issues/102

Can we remove the pre-commit hook and let CI take care of that? Because This repo is mostly unusable on Windows in it's current state.

**Comment by andyfry01** [comment]
@hbowden this was broken for me too, although now I think it should be working. See [line 10](https://github.com/appleboy/react-recaptcha/blob/master/package.json#L10) and [11](https://github.com/appleboy/react-recaptcha/blob/master/package.json#L11) in package.json. Previously pre commit was running test with a `--watch` flag, which was stalling the commit. Now pre commit runs test without `--watch`, so I think it will work for you.

**Comment by hbowden** [comment]
@andyfry01 No, this is still very much broken on Windows. Any ideas @appleboy ?

**Comment by appleboy** [comment]
I think we can remove the pre-commit hooks since we can just run the testing and lint in Travis CI.
